sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -761,6 +762,39 @@ export class JsonRpcProvider extends Provider {
     }
   }
 
+  //Dynamic Fields
```

Comment:
```suggestion
  // Dynamic Fields
```
nit: space after `//`